test/reporters/list.spec.js

```diff
@@ -98,7 +101,7 @@ describe('List reporter', function() {
       );
 
       Base.cursor = cachedCursor;
-      Base.symbols = cachedSymbols;
+      Base.symbols.ok = cachedOkSymbol;
```

Comment:
Additionally, I found original code didn't restore ok symbol because `Base.symbols.ok = expectedOkSymbol;` is changed refrenced `cachedSymbols`.

<img width="1013" alt="2018-10-24 4 22 33" src="https://user-images.githubusercontent.com/390146/47467382-fa62ef00-d7aa-11e8-89c9-9452ad7182ad.png">

Now it fixed.

<img width="996" alt="2018-10-24 4 23 02" src="https://user-images.githubusercontent.com/390146/47467394-064eb100-d7ab-11e8-91cd-b6f6cef252c6.png">

Reply:
I had noticed this as well. But your implementation only fixes one aspect of a larger problem. Please remove your "fix" and let's open another issue for this. For example, the `Base.cursor` functions are also "twiddled" with and not restored properly. It just goes unnoticed because it's never _actually_ used again. All tests messing around with any `Base.exports.<symbol>` should be deep cloning prior to any "twiddling".